source/known-limitations.rst

```diff
@@ -0,0 +1,28 @@
+.. _psa-known-limitations:
+
+================================================================================
+Percona Scheduler Admin tool known limitations
+================================================================================
+
+
+.. _do-not-combine:
+
+Do not combine the following options
+---------------------------------------------------
+
+The following options are mutually exclusive. An attempt to run them together
+results in error.
+
+*  :ref:`psa-update-write-weight` and :ref:`psa-auto-assign-weights`
+
+*  ``--write-node`` and :ref:`psa-auto-assign-weights`
```

Comment:
`--write-node` does not have a hyperlink.

Reply:
--write-node is not listed separately on the Options page, therefore, there is no anchor to link to.